### DISCUSSION TITLE
Any way to add chiadog's bot to a telegram group?

### DISCUSSION THREAD

**Discussion by rHilkner** [martomi/chiadog#260] [Q&A]
Hey guys, I'm using chiadog for a while and it's working great to send me private messages on Telegram, but I'd like to share this info with another person that farms with me. Is there any way to add the bot that I created with @BotFather to a group and make it send the notifications there? I added @IDBot to this group, got the telegram group-id and set it up in chiadog's config.yaml, but while testing the bot (with the testunit) it failed on all tests. Do you know if there's anything I'm missing on how to add the bot to a group or if this really is not implemented? Thanks!

**Comment by pieterhelsen** [comment]
From a quick google search:

when creating your group, just mention the full bot name (eg. @UniversalAgent1Bot) and it will list it as you type. Then you can just tap on it to add it.

Original answer: https://stackoverflow.com/a/40175742/1045783

**Reply by rHilkner** [reply]
Yes, I did that. The bot is in the group, but for some reason it's not being able to send messages there, only in the private chat.

**Reply by pieterhelsen** [reply]
You'll need to update the chat id I would assume:
https://github.com/martomi/chiadog/blob/main/INTEGRATIONS.md#telegram

You'll probably only be able to receive alerts in one channel (either your own personal chat or the group chat), based on that chat id.
